lib/util/support.js

```diff
@@ -139,8 +166,13 @@ function isSupported() {
 
   const rebrandedChrome = rebrandedChromeBrowser(browser);
   const mobileWebKit = mobileWebKitBrowser(browser);
-  const supportedMobileBrowsers = /android/.test(navigator.userAgent.toLowerCase()) ?
-    SUPPORTED_ANDROID_BROWSERS : SUPPORTED_IOS_BROWSERS;
+  const supportedMobileBrowsers = /android/.test(
+    navigator.userAgent.toLowerCase()
+  )
+    ? SUPPORTED_ANDROID_BROWSERS
+    : isIosWithThirdPartyWebRTCSupport()
+      ? SUPPORTED_IOS_THIRD_PARTY_BROWSERS
+      : SUPPORTED_IOS_BROWSERS;
```

Comment:
Feels like this is getting nested enough to warrant a block of code instead of a long expression.